substrate/client/network/src/request_responses.rs

```diff
@@ -684,7 +795,7 @@ impl NetworkBehaviour for RequestResponsesBehaviour {
 						// Received a request from a remote.
 						request_response::Event::Message {
 							peer,
-							message: Message::Request { request_id, request, channel, .. },
+							message: request_response::Message::Request { request_id, request, channel, .. },
```

Comment:
Revert
```suggestion
							message: Message::Request { request_id, request, channel, .. },
```

Reply:
I still cannot get rid of request_response (which is coming from libp2p) here. this is because `ev` being matched using the `libp2p` method `ToSwarm::GenerateEvent()`. Removing `libp2p` from `request_responses` will way be more complex (I tried...).

Reply:
That's fine. I just meant you should revert the unrelated change adding `request_response` prefix. You also removed the use declaration of `Message`, that should also be reverted as unrelated change.